modules/repository/repo.go

```diff
@@ -240,16 +237,19 @@ func MigrateRepositoryGitData(ctx context.Context, u *user_model.User,
 
 // cleanUpMigrateGitConfig removes mirror info which prevents "push --all".
 // This also removes possible user credentials.
-func cleanUpMigrateGitConfig(configPath string) error {
-	cfg, err := ini.Load(configPath) // FIXME: the ini package doesn't really work with git config files
+func cleanUpMigrateGitConfig(ctx context.Context, repoPath string) error {
+	cmd := git.NewCommand(ctx, "remote", "rm", "origin")
+	// if the origin does not exist
+	stdout, _, err := cmd.RunStdString(&git.RunOpts{
```

Comment:
See the comment of `RunStdString`:

```
// RunStdString runs the command with options and returns stdout/stderr as string. and store stderr to returned error (err combined with stderr).
func (c *Command) RunStdString(opts *RunOpts) (stdout, stderr string, runErr RunStdError) {
```

So, failed command also returns `err`, and `err` contains Stderr output.